config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get "/studios", to: "studios#index"
+  get "/studios/:id", to: "studios#show"
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get "/movies/:id", to: "movies#show"
+
+  # I wanted to do the path /movies/:movie_id/actors/:actor_id but wasn't
+  # able to get it to work, not sure why. Guidance here would be appreciated.
+  post "/movies/:id", to: "actor_movies#create"
```

Comment:
The path you were trying to get to work looks good; this one also looks good. Do you remember what kind of error you were getting when it wasn't working? My guess is it had something to do with the `params[:actor_id]` in your form, but hard to tell from here.